docs/guides/observability/telemetry.adoc

```diff
@@ -55,11 +57,58 @@ See the https://www.keycloak.org/observability/tracing[Root cause analysis with
 
 You can override the global OpenTelemetry settings via the Tracing options if you want to export traces to a different OpenTelemetry collector.
 
-<@opts.printRelevantOptions includedOptions="telemetry-*">
+== Logs
+
+<@features.techpreview feature="opentelemetry-logs"/>
+
+NOTE: The OpenTelemetry feature (`opentelemetry`) needs to be turned on (by default).
+
+It is possible to export {project_name}'s logs to the OpenTelemetry collector and be managed by various logging backends that support OpenTelemetry.
+
+=== Enable Logs
+You can enable OpenTelemetry Logs via CLI as follows:
+
+<@kc.start parameters="--features=opentelemetry-logs --telemetry-logs-enabled=true"/>
+
+For more details about OpenTelemetry Logs functionality in Quarkus, see the https://quarkus.io/guides/opentelemetry-logging[Quarkus OpenTelemetry Logging guide].
```

Comment:
One more thing. Is it actually beneficial to the Keycloak users the link to the Quarkus guide? The guide seems more dev oriented, it's not really focused on generic OTEL logs.